src/components/NewCommentForm.tsx

```diff
@@ -58,16 +60,27 @@
     }
 
     setSubmitting(true);
+    setIsLoading(true);
+
+    try {
+      await onSubmit({ name, email, body });
+      clear();
+    } finally {
+      setSubmitting(false);
+      setIsLoading(false);
+    }
```

Comment:
The `isLoading` state is set but not utilized in the UI. If you want to show a loading spinner or some indication of loading, consider using this state in your component's render logic.